syntax = "proto3";

option go_package = "tiaoyuyu.com/pb/gamepb";
option java_multiple_files = true;
option java_package = "gamepb";
option java_outer_classname = "gamepb";

package gamepb;

enum ProtocolType {
    EC2SPing = 0;
    ES2CPING = 1;
    EC2SGobangStep = 2;
    ES2CGobangStep = 3;
}